manual/statements/built/README.md

```diff
@@ -1,5 +1,268 @@
+condition, and additional options if necessary:
+
+```java
+BuiltStatement updateStatement = QueryBuilder.update("test").with(set("test", 1))
+        .where(eq("id", 1));
+```
+
+Update operation is passed as an argument to the `with` function, and additional
+operations could be chained via `and` calls:
+
+```java
+BuiltStatement updateStatement = QueryBuilder.update("test").with(set("t", "test 1"))
+        .and(set("x", 10)).where(eq("id", 1));
+```
+
+Besides the most often used `set` operation, there is a plenty of operations for work with
+all types of collections (lists, maps & sets): `add`, `discard`, `prepend`, `put`,
+`remove`, `setIdx`, etc.  For full list see the [QueryBuilder]'s documentation.
+
+To update counters you can use the `incr` & `decr` functions that takes the column name,
+and value by which column will be increased or decreased:
+
+```java
+BuiltStatement query = QueryBuilder.update("counters")
+        .with(incr("counter", 1)).where(eq("id", 1));
+```
+
+Similarly to insert statements, it's also possible to perform conditional update by
+calling either `ifExists` to perform only if entry exists, or calling `onlyIf` with
+`Clause` object - in this case row will be updated only if clause returns true:
+
+```java
+Statement updateStatement = QueryBuilder.update("test").with(set("t", "test 1"))
+        .where(eq("id", 1)).ifExists();
+```
+
+Setting the TTL & write timestamp is done the same way as described in previous section.
+
+### Deleting data
+
+You can delete either the whole row matching your condition:
+
+```java
+BuiltStatement deleteStmt = QueryBuilder.delete().from("test")
+                              .where(eq("id", "1")).and(eq("txt", "test"));
+```
+
+or specify a list of columns to delete:
+
+```java
+BuiltStatement deleteStmt = QueryBuilder.delete().from("test")
+                              .where(eq("id", "1")).and(eq("txt", "test"));
+```
+
+Specification of conditions is similar to the other operations described above, including
+conditional delete with `ifExists` & `onlyIf`.
+
+### Prepared statements
+
+If you're repeating the same operation very often, the more effective way will be to
+create the [prepared statement](../prepared/) from the `BuiltStatement`. To do it, instead
+of the real values, use the bind marker that is created either by calling of `bindMarker`
+that generates unnamed placeholder, or by calling `bindMarker("name")` to create a named
+placeholder.  After the statement is generated, just prepare it as usual, then bind &
+execute:
+
+```java
+BuiltStatement selectOne2 = QueryBuilder.select().from("test", "test")
+        .where(eq("id", bindMarker()));
+PreparedStatement preparedStatement = session.prepare(selectOne2);
+ResultSet rs = session.execute(preparedStatement.bind(1));
+```
+
+### Setting additional options
+
+As in the case of the usual statements, you can also set additional options for given
```

Comment:
Suggestion: "As it is the case for regular statements, you can also set options on built statements such as the consistency level..."